lms/djangoapps/instructor/enrollment.py

```diff
@@ -290,27 +289,29 @@ def _reset_module_attempts(studentmodule):
 
 def _fire_score_changed_for_block(course_id, student, block, module_state_key):
     """
-    Fires a PROBLEM_SCORE_CHANGED event for the given module. The earned points are
-    always zero. We must retrieve the possible points from the XModule, as
-    noted below.
+    Fires a PROBLEM_RAW_SCORE_CHANGED event for the given module.
+    The earned points are always zero. We must retrieve the possible points
+    from the XModule, as noted below.
     """
     if block and block.has_score:
```

Comment:
Nit: You can significantly simplify the readability and logic of this code if you restructure it.  Hint: with only 1 `return` statement.